common/v2/components/TransactionFlow/TransactionReceipt.scss

```diff
@@ -59,6 +72,21 @@
           font-size: 16px;
         }
       }
+      &-zapInfo {
+        display: flex;
+        justify-content: flex-end;
+        align-items: flex-start;
+        img {
+          margin-right: 10px;
+          @media (max-width: 550px) {
+            display: none;
+          }
+        }
+        .Amount .Amount-fiat{
+          color: #B5BFC7;
+          font-size: 16px;
+        }
+      }
```

Comment:
If you really must use scss, please create a new className.
This method of nesting name declarations is really difficult to maintain.